src/PrestaShopBundle/Form/Type/StyledCheckboxType.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Form\Type;
+
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+
+/**
+ * Class StyledCheckboxType
+ */
+class StyledCheckboxType extends CheckboxType
```

Comment:
I'm not sure you need to create a new FormType for this, you can try adding an attribute `material_design` Either in your `PartialRefundType ` like here https://github.com/PrestaShop/PrestaShop/blob/6d2f2405d6839357fe7dd5aa92144c5f63f4d515/src/PrestaShopBundle/Form/Admin/Improve/International/Currencies/CurrencyType.php#L104
Or inside the twig template https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/messages.html.twig

Although it seems weird that we need to explicitly add this, it should be somehow automatic

Reply:
It seems this class is not longer used but it's still in the PR